README.md

```
@@ -23,3 +23,36 @@ make dev
 ```shell
 poetry run scrapy runspider contrib/scraper_natal.py -o natal.jsonl
 ```
+
+#### WhatsApp Automation
+The WhatsApp automation is still a very manual process. There are two scripts:
+- `contrib/whats-extract.js`: extracts the messages from WhatsApp, format them,
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)